src/server.js

```diff
@@ -22,6 +22,9 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
 // Routes
+app.get('/', (req,res) => {
+  res.send('welcome to my app!')
+})
 app.use(authRoutes);
 app.use(v1Routes);
 app.use(v2Routes);
```

Comment:
You can change to `app.use('/api/v2',v2Routes);` this way, you can remove the `/api/v2` part from your v2 routes because this will do it for you.